source/documentation/data-docs/amazon-s3.md

```diff
@@ -228,10 +230,10 @@ You can find out more about how to use `s3browser` on [GitHub](https://github.co
 
 #### `pandas`
 
-You can use any of the `pandas` read functions (for example, `read_csv` or `read_json`) to download data directly from Amazon S3. This requires that you have installed the `s3fs` package. To install the `s3fs` package, run the following code in a terminal:
+You can use any of the `pandas` read functions (for example, `read_csv` or `read_json`) to download data directly from Amazon S3. This requires that you have installed the `pandas` and `s3fs` packages. To install these, run the following code in a terminal:
 
 ```
-pip install s3fs
+pip install pandas s3fs
```

Comment:
Could you also point users to [`aws-wrangler`](https://aws-data-wrangler.readthedocs.io/en/stable/tutorials/003%20-%20Amazon%20S3.html)? For a `pip install` on AP I think it is better to do `python -m pip ...`

Reply:
Thanks, I've made the suggested changes.